PostHog/Autocapture/UIView+PostHogLabel.swift

```diff
@@ -0,0 +1,29 @@
+//
+//  UIView+PostHogLabel.swift
+//  PostHog
+//
+//  Created by Yiannis Josephides on 04/12/2024.
+//
+
+#if os(iOS) || targetEnvironment(macCatalyst)
+    import UIKit
+
+    public extension UIView {
+        /**
+         Adds a custom label to this view for use with PostHog's auto-capture functionality.
+
+         By setting a custom label, you can easily identify and filter interactions with this specific element in your analytics data.
+
+         ### Usage
+         ```swift
+         let myView = UIView()
+         myView.postHogLabel = "customLabel"
+         ```
+         */
+        var postHogLabel: String? {
```

Comment:
I think a follow up PR would be to read the `postHogLabel` in session recording and if its `ph-no-capture`, we mask it, or adding a new label for it, so we have another option instead of accessibility labels, create an issue?